docs/src/main/asciidoc/Bootable_Guide.adoc

```diff
@@ -0,0 +1,197 @@
+
+When the bootable JAR Maven plugin builds a JAR, WildFly feature-packs are retrieved and 
+their content is assembled to create the server contained in the JAR.
+
+Once you have identified a WildFly Galleon feature-pack, you need to select a set of 
+link:#wildfly_defined_galleon_layers[WildFly Galleon layers] that are used to compose the server.
+
+The set of Galleon layers to include is driven by the application you are developing. 
+The list of link:#wildfly_defined_galleon_layers[WildFly Galleon layers] provides details on 
+the server features that each layer brings. Make sure that the API and server features you are using (eg: JAX-RS, Eclipse Micropfile Config, datasources) are brought by the 
+layers you are choosing.
+
+If you decide to not specify Galleon layers, a server containing all Eclipse Microprofile subsystems is 
+provisioned (the server configuration is identical to the ```standalone-microprofile.xml``` default one).
+
+Maven Plugin configuration extract example:
+
+[source,xml]
+----
+<build>
+  <plugins>
+    <plugin>
+      <groupId>org.wildfly.plugins</groupId>
+      <artifactId>wildfly-jar-maven-plugin</artifactId>
+      <configuration>
+        <feature-pack-location>wildfly@maven(org.jboss.universe:community-universe)</feature-pack-location> (1)
+        <layers>
+          <layer>jaxrs-server</layer> (2)
+        </layers>
+      </configuration>
+      <executions>
+        <execution>
+          <goals>
+            <goal>package</goal>
+          </goals>
+        </execution>
+      </executions>
+    </plugin>
+  </plugins>
+</build>
+----
+
+(1) In this plugin configuration extract, we are retrieving the latest WildFly Galleon feature-pack installed in the
+ ```org.jboss.universe:community-universe``` Galleon universe. In case you would like to provision a specific version of the server,
+you would need to specify the server version, for example ```wildfly@maven(org.jboss.universe:community-universe)#21.0.0.Final```
+
+(2) The link:#gal.jaxrs-server[jaxrs-server layer] and all its dependencies are provisioned. 
+
+include::_galleon/Galleon_layers.adoc[]
+
+[[wildfly-jar-maven-plugin-additional-configuration]]
+== Additional configuration
+
+The plugin allows you to specify additional configuration items:
+
+* A set of WildFly CLI scripts to execute to fine tune the server configuration. 
+* Some extra content to be copied inside the bootable JAR (e.g.: server keystore).
+
+Check https://docs.wildfly.org/bootablejar[this documentation] for more details on how to configure execution of
+CLI scripts and to package extra content.
+
+[[wildfly-jar-maven-plugin-build]]
+== Packaging your application
+
+Call ``` mvn package```  to package both your application and the bootable JAR in the file ```<project build directory>/<project final name>-bootable.jar```
+
+In order to speed-up the development of your application (avoid rebuilding the JAR each time your application is re-comiled), 
```

Comment:
s/re-comiled/re-compiled

Reply:
Thank-you @yersan , I will update it once I got all feedback. I think the Layers table helps reveal the features a layers bring-in by being able to navigate the dependencies.